internal/interop/interopconnect/test_cases.go

```diff
@@ -374,7 +390,7 @@ func DoDuplicatedCustomMetadataUnary(t crosstesting.TB, client connectpb.TestSer
 		t,
 		client,
 		map[string][]string{
-			leadingMetadataKey: {leadingMetadataValue, leadingMetadataValue + ",more_stuff"},
+			leadingMetadataKey: {leadingMetadataValue, leadingMetadataValue + ";more_stuff"},
```

Comment:
Changed to a semicolon so that the tests didn't break, since it would split on this comma also. Note that in the wild, it is still possible to use commas in duplicate header values, but this is to make our testing lives easier.